### ISSUE TITLE
Subversion build error

### ISSUE THREAD

**Issue by apv174ru** [nono303/win-build-scripts#2]
```
D:\SDK\src\sqlite\sqlite3.c(34597,16): error C4013: sqlite3PagerWalSystemErrno: undefined; assuming "extern" returning "int" [D:\SDK\src\subversion\build\win32\vcnet-vcproj\libsvn_subr.vcxproj]
(compilation of source file "../../../subversion/libsvn_subr/sqlite3wrapper.c")
```
All sources downloaded via srccreate.bat, then
vs17 && x64 && avx 0 && quic 0 && go full
vs17 && x64 && avx 0 && quic 0 && go subversion

**Comment by nono303** [comment]
Hi @Dumonde,
Yes, I'm aware of this issue: `sqlite3.c` need `USE_SEH` config flag to expose `sqlite3PagerWalSystemErrno`
Better than (re)compiling `sqlite3.c` for `sqlite3wrapper.c`, I'm currently patching `gen_win_dependencies.py` to use shared sqlite3 dll
But as 1.14.x py scripts are not compatible with latest pyton3 _(patch from trunk had not been back-ported)_ I've to manage different patch for 1.14 & 1.15 (trunk)
With recent 1.14.3 release, I'll also refactor dependencies generation (`go full` will be replaced by `go subversion-deps`) for a fastest and easiest way to provide a clean bundle.
I'll commit these changes here under **tag 1.8.3** and release 1.14.3 on https://github.com/nono303/win-svn **ASAP**

**Comment by nono303** [comment]
I just pushed https://github.com/nono303/win-build-scripts/tree/1.8.3
I let you test it and give me your feedback
And if you can, don't hesitate to check & validate https://github.com/nono303/win-svn/tree/1.14.3 ;)

**Comment by apv174ru** [comment]
Yeah, the error is gone, thanks.
I've successfully built httpd, subversion and deps.
PS: Hardcoded path [/cygdrive/c/sdk/](https://github.com/nono303/win-build-scripts/blob/5781b548e0f7d1c78559ae2118f4cbb0ad45e585/modules/icu.bat#L10) gives error while building icu, coz my PATH_SDK_ROOT=D:\SDK. 
I've changed manually to **`/cygdrive/d/sdk`**

**Comment by nono303** [comment]
Oups... Missed this one, will fix it.
Many thx for your feedback!

**Comment by nono303** [comment]
> PS: Hardcoded path [/cygdrive/c/sdk/](https://github.com/nono303/win-build-scripts

Fixed in [1.8.4](https://github.com/nono303/win-build-scripts/releases/tag/1.8.4)
